charts/home-assistant/values.yaml

```diff
@@ -322,6 +322,18 @@ appdaemon:
     loadBalancerSourceRanges: []
     # nodePort: 30000
 
+esphome:
+  enabled: false
+  # Mount home-assistant's secrets.yaml into the esphome container.
+  extraVolumes:
+    - name: ha-config
+      persistentVolumeClaim:
+        claimName: home-assistant
```

Comment:
This is a little flaky as it depends on the release name but I'm not quite sure how to get around that.

Reply:
We probably need to find a clever way to handle this, as hard-coding to `home-assistant` is less-than-ideal.  I'm poking around about how to communicate a templated name (or equivalent) here instead.

Reply:
Perhaps using something like? Does the claim always match the release name?

https://github.com/billimek/billimek-charts/blob/master/charts/powerdns/templates/secret.yaml#L10

Reply:
I don't think templates can work inside the `values.yaml` file.  I came across https://github.com/helm/helm/issues/2492 describing pretty much this problem.

It seems that until there is a way to handle templates in the values, we are probably going to just need to hard-code, like we do here.

What do you all think?